src/signals/incident-management/index.js

```diff
@@ -18,6 +18,7 @@ import IncidentOverviewPage from './containers/IncidentOverviewPage';
 import { getDistricts, getFilters, searchIncidents, requestIncidents } from './actions';
 import IncidentDetail from './containers/IncidentDetail';
 import DefaultTextsAdmin from './containers/DefaultTextsAdmin';
+import IncidentSplitContainer from './containers/IncidentSplitContainer';
 import LegacyIncidentSplitContainer from './containers/LegacyIncidentSplitContainer';
```

Comment:
This container and its child components can be removed, right?

Reply:
Like discussed, it was unclear if the legacy one had to stay, done